clang/unittests/ASTMatchers/ASTMatchersNarrowingTest.cpp

```diff
@@ -2235,6 +2235,21 @@ TEST_P(ASTMatchersTest, ArgumentCountIs_CXXConstructExpr) {
                  Constructor1Arg));
 }
 
+TEST_P(ASTMatchersTest, hasDependentName_DependentScopeDeclRefExpr) {
```

Comment:
nit: looking at other tests in this file, the test names start with a capital letter (e.g. `HasSize_CXX`) even if the matcher they are testing starts with lowercase (e.g. `hasSize`)

Reply:
Done